themes/bootstrap3/templates/myresearch/mylist.phtml

```diff
@@ -87,8 +87,10 @@
     <div><strong><?=$this->transEsc('Tags')?>:</strong> <span class="list-tags"><?=implode(', ', array_map([$this, 'escapeHtml'], $listTags))?></span></div>
   <?php endif; ?>
   <?php if ($recordTotal > 0): ?>
-    <form class="form-inline" method="post" name="bulkActionForm" action="<?=$this->url('cart-myresearchbulk')?>" data-lightbox data-lightbox-onsubmit="bulkFormHandler">
+    <?php $multiPageSelection = $this->config()->get('config')['Social']['multi_page_favorites_selection'] ? ' multi-page-selection' : ''; ?>
```

Comment:
This could probably use some null coalescing in case the setting is unset, to avoid notices.

Reply:
Done